### DISCUSSION TITLE
MSCS%40WUSTL/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#70] [Announcements]
# MSCS%40WUSTL/

https://csmsapp.github.io/MSCS%40WUSTL/

**Comment by beckysx** [comment]
根据21fall入学的学生评价，找summer intern一片惨状，知道21fall入学的所有人找实习都很难，但是数据烂成这样的第一次见

**Reply by beckysx** [reply]
现在是22年4月28，计算机院200人找工，只有不到十个人找到实习，闻所未闻

**Reply by xichenpan** [reply]
呜呜 同样听说了，太离谱了。。

**Reply by xichenpan** [reply]
可能对于非top校，有个好的location能本地中小厂捞一捞还是比较重要的

**Reply by MuggleZzzH** [reply]
佬, 最近几年怎么样🥲

**Comment by llxblhyvia** [comment]
笑死 23fall录的两个 一个这个一个NYU MSIS 都是B- 这么看排名是真滴准 我也是真滴菜hhhhhh

**Comment by Zouzijian-hue** [comment]
还行
________________________________
发件人: MuggleZzzH ***@***.***>
发送时间: 2025年8月31日 6:01
收件人: opencsapp/opencsapp.github.io ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [opencsapp/opencsapp.github.io] MSCS%40WUSTL/ (Discussion #70)


佬, 最近几年怎么样🥲
